src/components/home/elements/TransactionRecords.vue

```diff
@@ -239,6 +243,16 @@ export default {
                             let senderAddr = recItem['proofs'] ? recItem['proofs'][0]['address'] : 'no sender'
                             recItem['index'] = ++count
                             recList[month].push(recItem)
+                            if (recItem['type'] === EXECUTE_CONTRACT_TX) {
+                                let tokenId = common.contractIDToTokenID(recItem['contractId'])
+                                if (certify.isCertified(tokenId) && (recItem['functionIndex'] === 4 || (recItem['functionIndex'] === 3 && base58.decode(recItem['functionData'])[1] === 2))) {
+                                    let functionData = convert.parseFunctionData(recItem['functionData'])
+                                    recItem['recipient'] = functionData.recipient
+                                    recItem['amount'] = functionData.amount
```

Comment:
The `parseFunctionData` should change to a common function that can parse any function parameters, not only send token. Once done, the `functionData.recipient` should change to `functionData[0]` and `functionData.amount` change to `functionData[1]`.